src/panels/config/ha-panel-config.ts

```diff
@@ -40,7 +45,82 @@ declare global {
 }
 
 export const configSections: { [name: string]: PageNavigation[] } = {
-  integrations: [
+  dashboard: [
+    {
+      path: "/config/integrations",
+      name: "Devices & Services",
+      description: "Integrations, devices, entities and areas",
```

Comment:
Translations are not added to these items, I expect us to tweak the content, it will be provided in a follow-up PR.